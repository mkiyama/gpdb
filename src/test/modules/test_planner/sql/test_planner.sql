-- start_matchsubs
--
-- # Ignore drop failures
-- m/.*extension \"test_planner\" does not exist/
-- s/(.*)//
--
-- # Remove all successful
-- m/INFO:  Success.*/
-- s/(.*)//
--
-- end_matchsubs

DROP EXTENSION test_planner;
CREATE EXTENSION test_planner;

SELECT test_planner();
DROP EXTENSION test_planner;
