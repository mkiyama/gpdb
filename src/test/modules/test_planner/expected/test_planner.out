-- start_matchsubs
--
-- # Ignore drop failures
-- m/.*extension \"test_planner\" does not exist/
-- s/(.*)//
-- 
-- # Remove all successful
-- m/INFO:  Success.*/
-- s/(.*)//
--
-- end_matchsubs
DROP EXTENSION test_planner;

CREATE EXTENSION test_planner;
SELECT test_planner();




 test_planner
--------------

(1 row)

DROP EXTENSION test_planner;
